common/data_refinery_common/models/documents.py

```diff
@@ -0,0 +1,88 @@
+from elasticsearch_dsl import analyzer
+
+from django_elasticsearch_dsl import DocType, Index, fields 
+
+from .models import Sample, Experiment, Organism
+
+experiment_index = Index('experiments')
+experiment_index.settings(
+    number_of_shards=1,
+    number_of_replicas=0
+)
+
+html_strip = analyzer(
+    'html_strip',
+    tokenizer="standard",
+    filter=["standard", "lowercase", "stop", "snowball"],
```

Comment:
I had to look up what `snowball` meant and according to [stack overflow](https://stackoverflow.com/a/9443368/6095378):

> Similarly snowball is a combination of stopword , lowercase and standard analyzer

So isn't this redundant?